search/presenter/src/main/java/com/jakewharton/sdksearch/search/presenter/SearchPresenter.kt

```diff
@@ -24,8 +23,7 @@ class SearchPresenter(
   private val _models = ConflatedBroadcastChannel<Model>()
   val models: ReceiveChannel<Model> get() = _models.openSubscription()
 
-  private val _events = PublishRelay.create<Event>()
-  val events: Consumer<Event> get() = Consumer { _events.accept(it) }
+  val _events = PublishRelay.create<Event>()
```

Comment:
This exposes the implementation details to external callers which is incorrect. They should only be able to push in events, not subscribe to them in any way.